app/views/editor/thang/hslCalculator.vue

```diff
@@ -0,0 +1,38 @@
+<script>
```

Comment:
This is a simple color calculator to help translate the hex colors to the hsl that we use.